Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -555,34 +555,13 @@ public void Init(IEOSCoroutineOwner coroutineOwner)
                 Init(coroutineOwner, EOSPackageInfo.ConfigFileName);
             }
 
-            //-------------------------------------------------------------------------
-            private EOSConfig LoadEOSConfigFileFromPath(string eosFinalConfigPath)
-            {
-                string configDataAsString = "";
-#if UNITY_ANDROID && !UNITY_EDITOR
-                configDataAsString = AndroidFileIOHelper.ReadAllText(eosFinalConfigPath);
-#else
-                if (!File.Exists(eosFinalConfigPath))
-                {
-                    throw new Exception("Couldn't find EOS Config file: Please ensure " + eosFinalConfigPath +
-                                        " exists and is a valid config");
-                }
-
-                configDataAsString = File.ReadAllText(eosFinalConfigPath);
-#endif
-                var configData = JsonUtility.FromJson<EOSConfig>(configDataAsString);
-
-                print("Loaded config file: " + configDataAsString);
-                return configData;
-            }
-
             //-------------------------------------------------------------------------
             public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
             {
                 string eosFinalConfigPath = Path.Combine(Application.streamingAssetsPath, "EOS", configFileName);
                 if (loadedEOSConfig == null)
                 {
-                    loadedEOSConfig = LoadEOSConfigFileFromPath(eosFinalConfigPath);
+                    loadedEOSConfig = JsonUtility.FromJsonFile<EOSConfig>(eosFinalConfigPath);
```

Comment:
90% of the time we are doing a `FromJson` function, we are doing it immediately after reading the file. This helper method enables us to consolidate that behavior into one place, unifying the way we deal with things like IO and reducing points of failure.